.github/workflows/go.yml

```diff
@@ -12,6 +12,9 @@ jobs:
     env:
       FLAKY_REGEX: "ava-labs/libevm/(triedb/pathdb|eth|eth/tracers/js|eth/tracers/logger|accounts/abi/bind|accounts/keystore|eth/downloader|miner|ethclient|ethclient/gethclient|eth/catalyst)$"
     runs-on: ubuntu-latest
+    concurrency:
```

Comment:
TIL about `concurrency` configuration! Thank you.

Reading up on it, it turns out you can define this at the workflow level ([example](https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#example-using-concurrency-and-the-default-behavior)) so only one is necessary per file.

I think placing it immediately after the `on:` block makes it stand out more so any future copy-paste CI jobs are more likely to include it.

Reply:
Indeed plus defining this concurrency with workflow-ref as I did, more than once in the same workflow, will (as it did) stop one of the 2 jobs 😄 silly me